core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -160,6 +162,20 @@ class SparkConf(loadDefaults: Boolean) extends Cloneable with Logging {
     set("spark.serializer", classOf[KryoSerializer].getName)
     this
   }
+  /**
+   * Use Kryo serialization and register the given set of Avro schemas so that the generic
+   * record serializer can decrease network IO
+   */
+  def registerAvroSchemas(schemas: Schema*): SparkConf = schemas.foldLeft(this) { (conf, schema) =>
+    conf.set(avroSchemaKey(schema), schema.toString)
+  }
```

Comment:
can you start the body of the method on the next line, and wrap the entire method w/ braces?